tests/integration/test_kube_ovn.py

```diff
@@ -453,9 +509,14 @@ def parse_tc_show(stdout):
     return int(limit_value)
 
 
-async def annotate_obj(client, obj, annotation_dict):
+async def annotate_obj(client, obj, annotation_dict, patch_type=PatchType.STRATEGIC):
     log.info(f"Annotating {type(obj)} {obj.metadata.name} with {annotation_dict} ...")
     obj.metadata.annotations = annotation_dict
     client.patch(
-        type(obj), obj.metadata.name, obj, namespace=obj.metadata.namespace, force=True
+        type(obj),
+        obj.metadata.name,
+        obj,
+        namespace=obj.metadata.namespace,
+        force=True,
+        patch_type=patch_type,
```

Comment:
i think you can delete this -- looks like you replaced with a fixture

Reply:
ah yeah, thanks!